start

```diff
@@ -225,6 +225,10 @@ function init_db() {
 	chown postgres:postgres $PGDATA
 	chmod 0700 $PGDATA
 
+	# In some environments this directory is missing.
+	mkdir -p ~/../var/run/postgresql
```

Comment:
❓ The `~/..` is interesting. Why is the relative to home used? That would suggest with a typical linux home directory setup that we hope to write this to the `/home/var/run/postgresql` folder, or with a setup where the user's home is one above root that it is simply `/var/run/postgresql` which I think is the common location for `/var/run` paths. If the goal is to write to the `/var/run/postgresql` folder, we can drop the `~/...`.

Reply:
>  If the goal is to write to the /var/run/postgresql folder, we can drop the ~/....

ah cool I actually didn't know `cd /` points to the home dir, ya that was my intention that is simpler. TIL!
https://github.com/stellar/docker-stellar-core-horizon/pull/276/commits/301e0f2844b55759134fd560783c0a39f49cea26

Reply:
`cd /` changes directory to the root. Normally the car for lives in the root and not in a home directory.